python_modules/libraries/dagster-dbt/dagster_dbt/cloud/resources.py

```diff
@@ -181,6 +181,44 @@ def run_job(self, job_id: int, **kwargs) -> Dict[str, Any]:
         )
         return resp
 
+    def get_runs(
+        self,
+        include_related: Optional[List[str]] = [],
```

Comment:
The default value here should be None instead of empty list (see: https://satran.in/b/python--dangerous-default-value-as-argument, but more immediately this causes a linting error)

Reply:
I had never seen that before but makes sense, thanks for the article